core/store/models/job_spec.go

```diff
@@ -213,10 +215,26 @@ type InitiatorParams struct {
 	ToBlock    *utils.Big        `json:"toBlock,omitempty" gorm:"type:varchar(255)"`
 	Topics     Topics            `json:"topics,omitempty" gorm:"type:text"`
 
-	RequestData JSON    `json:"requestData,omitempty" gorm:"type:text"`
-	Feeds       Feeds   `json:"feeds,omitempty" gorm:"type:text"`
-	Threshold   float32 `json:"threshold,omitempty" gorm:"type:float"`
-	Precision   int32   `json:"precision,omitempty" gorm:"type:smallint"`
+	RequestData     JSON     `json:"requestData,omitempty" gorm:"type:text"`
+	Feeds           Feeds    `json:"feeds,omitempty" gorm:"type:text"`
+	Threshold       float32  `json:"threshold,omitempty" gorm:"type:float"`
+	Precision       int32    `json:"precision,omitempty" gorm:"type:smallint"`
+	PollingInterval Duration `json:"pollingInterval,omitempty"`
+}
+
+// FluxMonitorDefaultInitiatorParams are the default parameters for Flux
+// Monitor Job Specs.
+var FluxMonitorDefaultInitiatorParams = InitiatorParams{
+	PollingInterval: Duration(time.Minute),
+}
+
+// SetDefaultValues returns a InitiatorParams with empty fields set to their
+// default value.
+func (i *InitiatorParams) SetDefaultValues(typ string) {
+	if typ == InitiatorFluxMonitor {
+		err := mergo.Merge(i, &FluxMonitorDefaultInitiatorParams)
+		logger.PanicIf(err)
```

Comment:
This panic is a bit concerning, if you believe this code to be infallible (i.e. shouldn't return an err) then it'd be helpful to have a comment why. Otherwise this should return error.